pkg/csi/controllerserver.go

```diff
@@ -661,26 +661,53 @@ func (cs *controllerServer) DeleteSnapshot(ctx context.Context, req *csi.DeleteS
 	return &csi.DeleteSnapshotResponse{}, nil
 }
 
+func getVolumeType(pv *v1.PersistentVolume) (volumeType string, err error) {
+	// Get volumeType
+	volumeType = string(pkg.VolumeTypeUnknown)
+	if pv != nil && pv.Spec.CSI != nil {
+		if value, ok := pv.Spec.CSI.VolumeAttributes["volumeType"]; ok {
```

Comment:
Use defined global variable 'VolumeTypeTag' instead of hard code "volumeType"

Reply:
done